tasks/analysis.py

```diff
@@ -435,11 +436,29 @@ def process_impact_result(self, impact_result, analysis_id):
             # pass on template path according to headless service
             custom_template_path = headless_template_path
 
+        # define the layer order of the map report
+        layer_order = (
+            list(settings.LAYER_ORDER) if settings.LAYER_ORDER else None)
+        if layer_order:
+            layer_source = {
+                'impact': impact_url,
+                'hazard': get_layer_path(analysis.hazard_layer),
+                'exposure': get_layer_path(analysis.exposure_layer)
+            }
+            if analysis.aggregation_layer:
+                layer_source.update({
+                    'aggregation': get_layer_path(analysis.aggregation_layer)})
+            else:
+                layer_order.remove('@aggregation')
+
+            layer_order = substitute_layer_order(layer_order, layer_source)
```

Comment:
Do we already include this function?

Reply:
yes we did, it turns out that I copied the whole line of helpers file in realtime to geosafe :joy:

Reply:
Lol, I was doing the same thing with this so I added the functions on utils twice!